containers/compose-base.yaml

```diff
@@ -105,12 +105,12 @@ services:
       - ../.env
     ports:
       - ${POSTGRES_PORT}:${POSTGRES_PORT}
-    user: "${HOST_UID:-1000}:${HOST_GID:-1000}"
     environment:
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
       POSTGRES_USER: ${POSTGRES_USER}
       PGDATA: /var/lib/postgresql/data/pgdata
     volumes:
+      - ${POSTGRES_CONFIG_FILE_HOST}:/etc/postgresql/postgresql.conf:ro
```

Comment:
maybe you need to add this variable to .env.tpl and to CI:
https://github.com/thegraphnetwork/EpiGraphHub/pull/189/checks#step:11:1180